README.md

```diff
@@ -129,15 +130,49 @@ cargo fmt --all
 cargo clippy --workspace --all-features -- -D warnings
 ```
 
+## Release Process
+
+### Overview
+
+We make releases within the `rust` repository specific to the crate(s) that will be consumed (e.g. agents@1.0.0, configuration@1.0.0, accumulator@1.0.0, etc).
+
+We follow [Semantic Versioning](https://semver.org/), where breaking changes constitute changes that break agent configuration compatibility.
+
+Releases are managed on GitHub [here](https://github.com/nomad-xyz/rust/releases).
+
+### Aggregating Release Notes
+
+- Want to aggregate list of all changes since last release
```

Comment:
is this a procedural step? 😂

Reply:
Lol no just specified intent, can delete that